frontend/src/alltypes.d.ts

```diff
@@ -41,3 +41,8 @@ declare module '@carbon/icons-vue/es/menu/20';
 
 // xLarge
 declare module '@carbon/icons-vue/es/search--locate/32';
+
+declare module '@carbon/icons-vue/es/*' {
+    const CarbonIcon: any;
+    export default CarbonIcon;
+}
```

Comment:
it does seem like that, but i didn't wanna risk messing up other component. Let me try removing them and see

Reply:
yup they still work, changed here https://github.com/bcgov/nr-forests-access-management/pull/1607/commits/089e31e9b4cb20f18f10f1e6c93de27f0e8c3871

`declare module '@carbon/icons-vue/es/*` informs TypeScript that this module exists, preventing errors about missing exports. This is necessary because `@carbon/icons-vue` doesn’t provide type definitions by default.